src/hooks/api/user.ts

```diff
@@ -149,10 +150,46 @@ const useGetRolesQuery = (
     ...configuration,
   });
 
+type SearchUser = {
+  uuid: string;
+  email: string;
+  username: string;
+};
+
+const findUserWithEmail = async ({
```

Comment:
To be consistent with other queries, I would use the following name
```suggestion
const getUserByEmail = async ({
```